{{cookiecutter.folder_name}}/Makefile

```diff
@@ -0,0 +1,78 @@
+# To make the document with the default settings:
+# > make
+#
+# To make with a specific format:
+# > make <format>
+#
+# To tar up a flat version of a specific format:
+# > make <format> tar
+#
+# Alex Drlica-Wagner <kadrlica@fnal.gov>
+#
+
+# Primary file names
+main=main
+#filename={{ cookiecutter.folder_name }}
+outname=paper
+
+# LATEX environment variables
+export TEXINPUTS:=.:./texmf/styles/:./tables/:
+export BSTINPUTS:=.:./texmf/bib/:
+
+# Document class switcher flags
+aastex=\def\flag{aastex}
+revtex=\def\flag{revtex}
+mnras=\def\flag{mnras}
+
+# Submission flags
+arxiv=\def\flag{emulateapj}
+submit=${aastex}
+#draft=${aastex}\def\linenums{\linenumbers}
+draft=${aastex}
+
+# Files to copy when making tarball
+tardir=tmp
+figdir=./figures
+figures=$(figdir)/*.pdf
+tabdir=./tables
+tables=$(tabdir)/*.tex
+styles=./texmf/styles/*.{sty,cls}
+bibs=./texmf/bib/*.bst
+source=$(main).{tex,bbl,bib} macros.tex authors.tex acknowledgments.tex
+
+tarfiles=$(figures) $(tables) $(styles) $(bibs) $(source)
+
+all: export flag = ${draft}
+all: main copy
+
+copy: 
+	cp ${main}.pdf ${outname}.pdf
```

Comment:
For simplicity we might consider dropping this - and nor removing main.pdf in the clean step.